mergify/src/main/scala/org/typelevel/sbt/mergify/MergifyPlugin.scala

```diff
@@ -114,8 +115,8 @@ object MergifyPlugin extends AutoPlugin {
       .dependsOn(ThisBuild / mergifyGenerate)
       .value,
     githubWorkflowCheck := githubWorkflowCheck.dependsOn(ThisBuild / mergifyCheck).value,
-    ThisBuild / mergifyLabelPaths := {
-      val labelPaths = (ThisBuild / mergifyLabelPaths).value
+    internallyCalculatedMergifyLabelPaths := {
```

Comment:
You still need the `ThisBuild` :)
```suggestion
    ThisBuild / internallyCalculatedMergifyLabelPaths := {
```

Reply:
😎